apps/vr-tests-web-components/.storybook/main.cjs

```diff
@@ -43,8 +43,8 @@ module.exports = /** @type {import('../../../.storybook/main').StorybookBaseConf
     config.plugins = config.plugins ?? [];
```

Comment:
## [🕵 fluentui-web-components-v3 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28912&commitId=c50cfaaef93aab50f276a14fb3a747d2ba730ad1&env=prod&clientType=FLUENTUI)


  <details><summary><b>Radio</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Radio.Checked - Dark Mode.normal.chromium.png|0| Added|
|Radio.Checked.normal.chromium.png|0| Added|
|Radio.Default - Dark Mode.normal.chromium.png|0| Added|
|Radio.Default.normal.chromium.png|0| Added|
|Radio.Disabled - Dark Mode.normal.chromium.png|0| Added|
|Radio.Disabled Checked - Dark Mode.normal.chromium.png|0| Added|
|Radio.Disabled Checked.normal.chromium.png|0| Added|
|Radio.Disabled.normal.chromium.png|0| Added|

</details>


  <details><summary><b>RadioGroup</b> 39 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|RadioGroup.Default - Dark Mode.1st selected.chromium.png|0| Added|
|RadioGroup.Default - Dark Mode.2nd selected.chromium.png|0| Added|
|RadioGroup.Default - Dark Mode.normal.chromium.png|0| Added|
|RadioGroup.Default - RTL.1st selected.chromium.png|0| Added|
|RadioGroup.Default - RTL.2nd selected.chromium.png|0| Added|
|RadioGroup.Default - RTL.normal.chromium.png|0| Added|
|RadioGroup.Default Checked.1st selected.chromium.png|0| Added|
|RadioGroup.Default Checked.2nd selected.chromium.png|0| Added|
|RadioGroup.Default Checked.normal.chromium.png|0| Added|
|RadioGroup.Default.1st selected.chromium.png|0| Added|
|RadioGroup.Default.2nd selected.chromium.png|0| Added|
|RadioGroup.Default.normal.chromium.png|0| Added|
|RadioGroup.Disabled - Dark Mode.1st selected.chromium.png|0| Added|
|RadioGroup.Disabled - Dark Mode.2nd selected.chromium.png|0| Added|
|RadioGroup.Disabled - Dark Mode.normal.chromium.png|0| Added|
|RadioGroup.Disabled Item - Dark Mode.1st selected.chromium.png|0| Added|
|RadioGroup.Disabled Item - Dark Mode.2nd selected.chromium.png|0| Added|
|RadioGroup.Disabled Item - Dark Mode.normal.chromium.png|0| Added|
|RadioGroup.Disabled Item.1st selected.chromium.png|0| Added|
|RadioGroup.Disabled Item.2nd selected.chromium.png|0| Added|
|RadioGroup.Disabled Item.normal.chromium.png|0| Added|
|RadioGroup.Disabled.1st selected.chromium.png|0| Added|
|RadioGroup.Disabled.2nd selected.chromium.png|0| Added|
|RadioGroup.Disabled.normal.chromium.png|0| Added|
|RadioGroup.Horizontal Stacked - RTL.1st selected.chromium.png|0| Added|
|RadioGroup.Horizontal Stacked - RTL.2nd selected.chromium.png|0| Added|
|RadioGroup.Horizontal Stacked - RTL.normal.chromium.png|0| Added|
|RadioGroup.Horizontal Stacked.1st selected.chromium.png|0| Added|
|RadioGroup.Horizontal Stacked.2nd selected.chromium.png|0| Added|
|RadioGroup.Horizontal Stacked.normal.chromium.png|0| Added|
|RadioGroup.Horizontal.1st selected.chromium.png|0| Added|
|RadioGroup.Horizontal.2nd selected.chromium.png|0| Added|
|RadioGroup.Horizontal.normal.chromium.png|0| Added|
|RadioGroup.Vertical - RTL.1st selected.chromium.png|0| Added|
|RadioGroup.Vertical - RTL.2nd selected.chromium.png|0| Added|
|RadioGroup.Vertical - RTL.normal.chromium.png|0| Added|
|RadioGroup.Vertical.1st selected.chromium.png|0| Added|
|RadioGroup.Vertical.2nd selected.chromium.png|0| Added|
|RadioGroup.Vertical.normal.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>